Lib/fontTools/designspaceLib/__init__.py

```diff
@@ -395,6 +662,183 @@ def map_backward(self, v):
         return piecewiseLinearMap(v, {v: k for k, v in self.map})
 
 
+class DiscreteAxisDescriptor(SimpleDescriptor):
+    """Container for discrete axis data.
+
+    Use this for axes that do not interpolate.
+
+    .. versionadded:: 5.0
+    """
+
+    flavor = "axis"
```

Comment:
I don't know, I've added it to new classes with the same value as the XML tag for the class (however I'm not using that field while reading or writing XML). I don't know if we can remove all of them, since it's not underscored, it's supposed to be part of the public API. Also, I notice that RuleDescriptor doesn't have one.

@LettError would you know about these `flavor` attributes and what their use is?